_posts/2021-05-17-changelog-147.markdown

```diff
@@ -0,0 +1,70 @@
+---
+layout: post
+title: "IntelliJ Rust Changelog #147"
+date: 2021-05-17 12:00:00 +0300
+---
+
+
+## New Features
+
+* [#7140] Enable [new name resolution engine](https://blog.jetbrains.com/clion/2020/12/intellij-rust-updates-for-2020-3/#new-resolve) by default
+
+* [#6456] Add WSL support. To use it, enable `org.rust.wsl` [experimental feature](https://plugins.jetbrains.com/plugin/8182-rust/docs/rust-faq.html#experimental-features)
```

Comment:
Let's also add a note that if there are both local Windows and WSL toolchains, the local one will be selected automatically. But it's possible to specify path via `\\wsl$` paths manually for now. And it will be improved in the future